.travis.yml

```diff
@@ -9,7 +9,7 @@ python:
 install:
     - pip install tox coveralls coverage
 script:
-    - tox -e py
+    - tox -e py,isort
```

Comment:
Here we might also want a run of the flake8 testenv, or is there any specific reason not to do so?